nshlib/nsh_dbgcmds.c

```diff
@@ -445,7 +445,8 @@ int cmd_hexdump(FAR struct nsh_vtbl_s *vtbl, int argc, FAR char **argv)
                       dumpbytes = count;
                     }
 
-                  snprintf(msg, sizeof(msg), "%s at %08x", argv[1], skip);
+                  snprintf(msg, sizeof(msg), "%s at %08zx", argv[1],
+                           (uintmax_t)skip);
```

Comment:
should we use ssize_t?

Reply:
i changed it to use uintmax_t consistently.

Reply:
consistent with what?

Reply:
i made printf format (%jx) and cast (uintmax_t) consistent.

ssize_t might or might not be wider than off_t.